ROMFS/px4fmu_common/init.d/airframes/50004_dfrobot_gpx_asurada

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+#
+# @name DF Robot GPX:Asurada
+
+#
+# @type Rover
+# @class Rover
+#
+# @board px4_fmu-v2 exclude
+#
+# @output MAIN2 Steering servo
+# @output MAIN3 Speed of left wheels
+# @output MAIN4 Speed of right wheels
+#
+# @maintainer Katrin Moritz
+#
+# @board px4_fmu-v2 exclude
+#
+
+sh /etc/init.d/rc.rover_defaults
+
+if [ $AUTOCNF = yes ]
+then
+	param set BAT_N_CELLS 2
+
+	param set EKF2_GBIAS_INIT 0.01
+	param set EKF2_ANGERR_INIT 0.01
+	param set EKF2_MAG_TYPE 1
+
+	param set FW_AIRSPD_MIN 0
+	param set FW_AIRSPD_TRIM 1
+	param set FW_AIRSPD_MAX 3
+
+	param set GND_THR_IDLE 0
+	param set GND_THR_CRUISE 0
+	param set GND_THR_MAX 0.5
+
+	# Because this is differential drive, it can make a turn with radius 0.
+	# This corresponds to a turn angle of pi radians.
+	# If a special case is made for differential-drive, this will need to change.
+	param set GND_MAX_ANG 3.142
+	param set GND_WHEEL_BASE 0.3
+
+	# TODO: Set to -1.0, to allow reversing. This will require many changes in the codebase
+	# to support negative throttle.
+	param set GND_THR_MIN 0.0
+	param set GND_SPEED_P 0.25
+	param set GND_SPEED_I 0.001
+	param set GND_SPEED_D 3
+	param set GND_SPEED_IMAX 0.125
+	param set GND_SPEED_THR_SC 1
+
+	param set MIS_LTRMIN_ALT 0.01
+	param set MIS_TAKEOFF_ALT 0.01
+
+	param set NAV_ACC_RAD 0.5
+
+	# Provide ESC a constant 1500 us pulse, which corresponds to
+	# idle on the Roboclaw motor controller on the Aion R1
```

Comment:
I'm confused how this one is related to the Roboclaw motor controller. Is this comment required?

Reply:
No, I think I have forgot to delete/change this comment.